poc/AWS_S3/env_settings.py

```diff
@@ -0,0 +1,13 @@
+from pydantic import BaseSettings
```

Comment:
os.getenv should do the job, do we need pydantic?

Reply:
os.getenv doesn't seem to work when I tried. Pydantic works and looks compatible with containers. 
https://pydantic-docs.helpmanual.io/usage/settings/